.github/workflows/build_docs.yml

```diff
@@ -5,19 +5,23 @@ name: Build PR Docs
 on:
   pull_request:
     branches:
-        - main
+      - main
   workflow_dispatch:
     inputs:
       make_target:
         description: "Enter make target: html html-noplot docs slimfast slimgallery"
         type: string
-        default: 'slimfast'
+        default: "slimfast"
   workflow_call:
     inputs:
       make_target:
         description: "Enter make target: html html-noplot docs slimfast slimgallery"
         type: string
-        default: 'slimfast'
+        default: "slimfast"
+      pr_ref:
```

Comment:
`pr_ref` or `pr_branch`?

Reply:
Ok, so this is analogous to the `github.ref` that would be normally used, which is using the `git` terminology. So i lean to keeping this the same, because it will make it easier for anyone diving into this, but keeping things consistent?